src/app/workflowitems-edit-page/advanced-workflow-action/advanced-workflow-action-select-reviewer/reviewers-list/reviewers-list.component.ts

```diff
@@ -92,6 +93,8 @@ export class ReviewersListComponent extends MembersListComponent implements OnIn
 
   selectedReviewers: EPerson[] = [];
 
+  searchCurrentMembersForm: UntypedFormGroup;
```

Comment:
Why did you add this form to the reviewers-list.component.ts?  It doesn't appear to be used at all?  Either it should be used or it should be removed.